### ISSUE TITLE
Zeppelin source code not found

### ISSUE THREAD

**Issue by mansilla** [SANSA-Stack/SANSA-Notebooks#13]
Hi,
when running `make` I'm getting this error

```
+ curl -fSL http://www-eu.apache.org/dist/zeppelin/zeppelin-0.7.2/zeppelin-0.7.2-bin-all.tgz -o /tmp/zeppelin.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (22) The requested URL returned error: 404 Not Found
ERROR: Service 'zeppelin' failed to build: The command '/bin/sh -c set -x     && curl -fSL "http://www-eu.apache.org/dist/zeppelin/zeppelin-0.7.2/zeppelin-0.7.2-bin-all.tgz" -o /tmp/zeppelin.tgz     && tar -xzvf /tmp/zeppelin.tgz -C /opt/     && mv /opt/zeppelin-* /opt/zeppelin     && rm /tmp/zeppelin.tgz' returned a non-zero code: 22
make: *** [up] Error 1
```

I think the source URL for Zeppelin needs a fix
`http://www-eu.apache.org/dist/zeppelin/zeppelin-0.7.2/zeppelin-0.7.2-bin-all.tgz`

**Comment by GezimSejdiu** [comment]
Hi @mansilla ,

the error seems strange to me. I'm aware that the zeppelin-0.7.x does not exist at that distro anymore, therefore I have [upgraded to 0.8.0](https://github.com/SANSA-Stack/SANSA-Notebooks/blob/develop/docker-compose.yml#L73). Is this comment related to the master branch of [bde2020/docker-zeppelin](https://github.com/big-data-europe/docker-zeppelin) or it is related to SANSA-Notebooks? I'm asking that because on our `make` command we just [download the `sansa-examples` bundled jar](https://github.com/SANSA-Stack/SANSA-Notebooks/blob/develop/Makefile#L1,L3).

Best regards,

**Comment by mansilla** [comment]
Hi Gezim. I received this error when following the [SANSA-Notebook readme instructions](https://github.com/SANSA-Stack/SANSA-Notebooks). So if the difference is just a `wget`, maybe this is an issue in both cases. But I'm not sure about this.

**Comment by GezimSejdiu** [comment]
um. could you please, let me know which version of bde-docker-zeppelin are you using (your local copy, maybe check it at the docker-compose file). I did remove the existing image from my computer and pull it again from the hub and there wasn't such issue. 

Not sure what is happening, maybe I'm missing something.

Best regards,

**Comment by GezimSejdiu** [comment]
Hi @mansilla ,

any new updates on this issue?

**Comment by mansilla** [comment]
Hi @GezimSejdiu. I have to confess, that I don't know what happened. But after I stop working on this a couple of days (a whole weekend), the error was fixed when I tried back. The mentioned error invoking a 404 URL (`http://www-eu.apache.org/dist/zeppelin/zeppelin-0.7.2/zeppelin-0.7.2-bin-all.tgz`) somehow was fixed. Just for sanity I checked the logs trying to know what was going on with the Apache site, and found out that
`http://www-eu.apache.org/dist/zeppelin/zeppelin-0.8.0/zeppelin-0.8.0-bin-all.tgz` was the new URL called. My best guess is that one of the docker dependencies you are using just updated its source code. I'm gonna close this now. Thanks again.
